front/app/modules/commercial/custom_idea_statuses/admin/containers/index.tsx

```diff
@@ -104,15 +102,19 @@ const IdeaStatuses = () => {
 
   const sortableStatuses = useMemo(() => {
     if (!isNilOrError(ideaStatuses) && defaultStatus) {
-      return ideaStatuses.filter(
+      return ideaStatuses.data.filter(
         (status) => status.attributes !== defaultStatus.attributes
       );
     }
 
     return [];
   }, [defaultStatus, ideaStatuses]);
 
-  if (!isNilOrError(ideaStatuses) && defaultStatus) {
+  if (isLoading) {
+    return <Spinner />;
+  }
+
+  if (ideaStatuses && defaultStatus) {
```

Comment:
There is no longer a need to check for `isNilOrError` -> data is either present or `undefined`.